core/arch/arm/plat-synquacer/rng_pta.c

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2018, Linaro Limited
+ */
+
+#include <io.h>
+#include <kernel/delay.h>
+#include <kernel/pseudo_ta.h>
+#include <mm/core_memprot.h>
+#include <string.h>
+#include <rng_pta.h>
+
+#define PTA_NAME "rng.pta"
+
+#define ENTROPY_POOL_SIZE	4096
+
+static uint8_t entropy_pool[ENTROPY_POOL_SIZE] = {0};
```

Comment:
I suppose you haven't been running checkpatch on this patch-set so far? I believe that checkpatch would complain about `{0}` (which I think should be `{ 0 }`). I've seen other places further down where I believe that checkpatch would complain.

Reply:
I have run checkpatch on this patch-set before. Also I don't see travis giving any checkpatch warnings or errors.